ext/dba/dba.c

```diff
@@ -966,6 +966,9 @@ PHP_FUNCTION(dba_fetch)
 			Z_PARAM_LONG(skip)
 			Z_PARAM_RESOURCE(id);
 		ZEND_PARSE_PARAMETERS_END_EX(goto standard;);
+
+		zend_error(E_DEPRECATED, "Calling dba_fetch() with $dba at the 3rd parameter is deprecated",
+			ZSTR_VAL(fbc->common.function_name));
```

Comment:
Passing that char* seems odd?

Reply:
Ah yes, there used to be a `%s` instead of `dba_fetch()`, and I forgot to remove the parameter.